### PR TITLE
Created Individual Unit files, PLUS up-to-date docs folder

### PR DESCRIPTION
**Pull request by warpdragon**
I merged everything I've got into this, docs, units, etc. Close the moar-docs pull request since this one has all of that stuff plus more.

### CODE REVIEW

File: game/scripts/npc/abilities/warlock_rain_of_chaos.txt~ [hunk 1]
```diff
@@ -0,0 +1,101 @@
+"warlock_rain_of_chaos"
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/warlock_rain_of_chaos.txt~]: this is a backup file from vim or notepad++, delete pleeeeease

File: docs/working_with_kv_files.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+When making KV files, you should always start with a valid, up-to-date template, many of which can be found at www.ModDota.com. 
+
+### Compile
+When you think you've got your files the way you want, run Node.js and gulp. This generates the file used by the game.
+
+### Debug
+Copy the contents of the KeyValue file you wish to debug into arhowk.github.io. Fix any errors in the *individual KV files* not in the compiled file. Once you've fixed all the errors, gulp again. This will put the updated KV files into the compiled form again.
+
+### Test in aaa
+Launch Dota 2 Tools; Load "aaa" addon; Load Console by pressing "~"; Launch the addon using the following command:
```
Comment (chrisinajar) [inline docs/working_with_kv_files.md]: aaa -> oaa

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Moar-Docs
+The goal of the Moar-docs branch is to compile EVERYONES information into one spot where I (and others) can view, edit, and finally merge everything into the final project. We are interested in "official" information directly related to the development, release, and support of the AAA project. Think of it this branch as a Library, ensuring that everyone has easy access to the most current information available. To that end, those who contribute are also asked to seek out and make contacts on the various teams, and work with key members to ensure that notes are taken and reported to Moar-docs whenever ideas or designs become "official".
```
Comment (chrisinajar) [inline docs/README.md]: aaa -> oaa

File: docs/github-for-noobs.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+# Getting Started
+
+## Setup
+
+First, follow the [Installation Process](/docs/install.md) but stop when you get to the part that says "Get the Addon". Now, instead of just Cloning (copying the angelarenaallstars/oaa repository) you'll create a Fork of it and work from there.
+If you do not already have a GitHub.com Account, create one [here](https://GitHub.com/join).
+
+## Forking
+Forking creates a simultaneous instance of aaa within your own GitHub repository.
```
Comment (chrisinajar) [inline docs/github-for-noobs.md]: aaa -> oaa

File: docs/quick_kv_updating_testing.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# Quick KV Updating/Testing
+
+## Setting up the Game
+1. Ensure you have the proper Clone of aaa. Check that you have the following:
```
Comment (chrisinajar) [inline docs/quick_kv_updating_testing.md]: aaa -> oaa

File: docs/russian/GitHub для чайников.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+Для начала, следуйте инструкции по [этой ссылке](/docs/install.md) , но остановитесь, когда дойдёте до части, под названием "Get the Addon". Теперь, вместо использования Cloning (копирования хранилища angelarenaallstars/oaa) вам надо создать его Fork и работать с ним (с Fork'ом).
+Если у вас ещё нет аккаунта на GitHub.com , то создайте его [тут](https://GitHub.com/join) .
+
+## Создание и работа с Fork'ами:
+
+Когда вы создаёте Fork , то в вашем хранилище на GitHub появляется копия хранилища, которое вы Fork'нули  (Иными словами, вы сделали себе копию проекта, над которым вы работаете и теперь он вам доступен на вашей странице/хранилище).
+Ваши шаги для создание Fork'а AngelArenaAllstars:
+-Перейдите на angelarenaallstars/oaa
+-Нажмите кнопку "Fork"
+-Выберите куда сохранять ваш Fork (Например: Fork angelarenaallstars/oaa to YOURUSERNAME/aaa).
```
Comment (chrisinajar) [inline docs/russian/GitHub для чайников.md]: aaa -> oaa

**Comment by Chronophylos** [comment]
Meaningful commits....

**Comment by Chronophylos** [comment]
Can you merge AngelArenaAllstars:master into your branch? `.gitignore` got updated.

**Comment by chrisinajar** [comment]
This needs to be merged with master, it conflicts now

**Pull request merged**